config/mappings/types_to_genres.yml

```diff
@@ -215,6 +215,76 @@ Video:
         uri: http://id.loc.gov/authorities/genreForms/gf2011026276
         source:
           code: lcgft
+Music:
```

Comment:
Changes to `config/mappings/` are tested in the existing `types_generator_spec.rb`, a dynamic spec which did not need updating in this PR.